CrossPare/src/main/java/de/ugoe/cs/cpdp/loader/JitDataLoader.java

```diff
@@ -0,0 +1,255 @@
+package de.ugoe.cs.cpdp.loader;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import de.ugoe.cs.cpdp.IParameterizable;
+import weka.core.Attribute;
+import weka.core.DenseInstance;
+import weka.core.Instances;
+
+/**
+ * Loader for just-in-time data
+ * @author jvdmosel
+ */
+public class JitDataLoader implements SingleVersionLoader, IBugMatrixLoader, IParameterizable {
+	
+	/**
+	 * the bug matrix
+	 */
+	private Instances bugMatrix = null;
+	
+	/**
+	 * the used label
+	 * default is jira
+	 */
+	private boolean adhoc = false;
+	
+	/**
+	 * the committer dates
+	 */
+	private ArrayList<OffsetDateTime> committerDates = new ArrayList<>();
```

Comment:
Should be:
private List<OffsetDateTime> committerDates = new ArrayList<>();

Reply:
private List<OffsetDateTime> committerDates = new ArrayList<>();

sorry, forgot the type of the generic in the comment